packages/contracts/contracts/Unirep.sol

```diff
@@ -113,6 +117,30 @@ contract Unirep is IUnirep, zkSNARKHelper, VerifySignature {
         );
 
         maxEpochKey = uint256(2)**config.epochTreeDepth - 1;
+        uint256 zero = 0;
+        uint256 one = 1;
+        uint256 defaultUSTLeaf = Poseidon5.poseidon(
```

Comment:
We can precompute default UST value.

Reply:
I don't think the savings are worth the complexity of passing another constructor argument, or hardcoding the hash value. We're planning to deploy to L2 networks so this gas cost is basically free.